addons/victoria-metrics-cluster/values.schema.json

```diff
@@ -32,15 +32,55 @@
         "WipeOut"
       ]
     },
-    "replicas": {
+    "vminsert.replicas": {
```

Comment:
here is an example of schme json.
https://www.arthurkoziel.com/validate-helm-chart-values-with-json-schemas/

Please refine the schme accordingly and make sure it works as expected. ( validate replicas/cpu/memory/ for  each component)